script/output_data/output_data.go

```diff
@@ -39,7 +39,7 @@ func generate() error {
 
 		f, err := os.Open(path)
 		if err != nil {
-			return err
+			continue
```

Comment:
What's going on here?

Reply:
this looks for files like `some_path/tests/valid/something.json` to feed some tests, which doesn't make sense for the health checker as doesn't accept any payload

Reply:
Could we check the processor type here and skip if it's the healthcheck? That way we'll still get the error for the other processors if something is wrong.